src/app/templates/settings-container.tpl

```diff
@@ -89,6 +89,10 @@
                     <div class="dropdown-arrow"></div>
                 </div>
             </span>
+            <span class="advanced">
+                <input class="settings-checkbox" name="defaultOsWindowFrame" id="defaultOsWindowFrame" type="checkbox" <%=(Settings.defaultOsWindowFrame? "checked='checked'":"")%>>
+                <label class="settings-label" for="defaultOsWindowFrame"><%= i18n.__("Default frame") %></label>
```

Comment:
Can we move this two positions down above 'Always On Top' and change to 'Native Title Bar' or something?
I know 'frame' is how it is in code but I think some users might not understand it.

Reply:
I find "Default frame" confusing too. IMO "Native window frame" or "Use native window frame" will be more clear.

Reply:
True, sorry missed out on second part of the comment.